src/xdgmenumaker

```diff
@@ -291,9 +291,11 @@ def icon_full_path(icon):
     # in non-standard directories
     if os.path.exists(icon):
         if max_icon_size:
+	    if os.path.splitext(icon)[1] == ".svg":
```

Comment:
You're right that we need to check for svg icons separately. But we need to discard svg icons altogether. I don't think any of the supported WMs can use svg icons! So, I see 2 problems with this:
1. We don't want so show svg icons. It should return None in the next line.
2. The indentation is wrong. You're missing one more space before the if check.